studioShared/schemas/documents/customerCase.ts

```diff
@@ -39,7 +39,34 @@ const customerCase = defineType({
       type: "internationalizedArrayString",
       title: "Title",
     }),
+    defineField({
```

Comment:
According to the design the colors for the title and badge should be the same, is there any way to validate this or should we add that in the description?

Reply:
Or could we use one color picker for the "main" color, and another for the text? 🤔

Reply:
If they should always be the same, then we should only have one field for it.